addon/helpers/compact.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember';
+
+const {
+  Helper,
+  isArray,
+  defineProperty,
+  get,
+  set,
+  observer,
+  isPresent,
+  computed,
+  copy,
+  typeOf,
+  A: emberArray
+} = Ember;
+const keys = Object.keys || Ember.keys;
```

Comment:
It is; but doesn't 1.13 still support IE 8, which needs the Ember.keys polyfill?